test/e2e-tests-auction.sh

```diff
@@ -55,4 +55,20 @@ step_with_retry "Account 4 has now 4 of token 1" \
 "mongo dfusion2 --eval \"db.accounts.findOne({'stateHash': '$EXPECTED_HASH'}).balances[121]\" | grep -w 4000000000000000000"
 
 step_with_retry "Account 3 has now 52 of token 0" \
-"mongo dfusion2 --eval \"db.accounts.findOne({'stateHash': '$EXPECTED_HASH'}).balances[90]\" | grep -2 52000000000000000000"
+"mongo dfusion2 --eval \"db.accounts.findOne({'stateHash': '$EXPECTED_HASH'}).balances[90]\" | grep -w 52000000000000000000"
+
+step_with_retry "[theGraph] Account 4 has now 4 of token 1" \
```

Comment:
Should we add this to the standing order e2e test as well?

Reply:
The standing order test doesn't really have anything to do with auction_settlement...